Obleak.Fody.Core/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Obleak.Fody.Core")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Obleak.Fody.Core")]
+[assembly: AssemblyCopyright("Copyright ©  2016")]
```

Comment:
Since this binary eventually deploys with NVivo client, might be worth adding some identifying details to this AssemblyInfo ? (I dunno what though)